sdk/environments/from_endpoint_test.go

```diff
@@ -6,7 +6,20 @@ import (
 )
 
 func TestFromEndpoint_Public(t *testing.T) {
-	actual, err := FromEndpoint(context.Background(), "https://management.azure.com")
+	actual, err := FromEndpoint(context.Background(), "https://management.azure.com", "")
+	if err != nil {
+		t.Fatalf("loading from endpoint: %+v", err)
+	}
+
+	if actual.Name != "AzureCloud" {
+		t.Fatalf("expected the Environment name to be `AzureCloud` but got %q", actual.Name)
+	}
+}
+
+func TestFromEndpoint_USGovernment(t *testing.T) {
+	t.Skip("USGovernment ARM metadata service has reverted to older schema")
```

Comment:
going to suggest we add a date to this error?

```suggestion
	t.Skip("Skipping since USGovernment ARM metadata service has reverted to older schema (2023-02-02)")
```